ui/preferences/src/main/java/de/danoeh/antennapod/ui/preferences/screen/downloads/DataFolderAdapter.java

```diff
@@ -44,11 +45,19 @@ public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
     public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
         StoragePath storagePath = entries.get(position);
         Context context = holder.root.getContext();
-        String freeSpace = Formatter.formatShortFileSize(context, storagePath.getAvailableSpace());
+        long availableSpace = storagePath.getAvailableSpace();
+        String freeSpace = Formatter.formatShortFileSize(context, availableSpace);
         String totalSpace = Formatter.formatShortFileSize(context, storagePath.getTotalSpace());
 
         holder.path.setText(storagePath.getPath());
-        holder.size.setText(String.format(freeSpaceString, freeSpace, totalSpace));
+        int quantity = (int) (availableSpace / (1024L * 1024L));
```

Comment:
- I think in the issue they wanted to use the freeSpace instead
- This does not always work. Your code assumes that megabytes are used, but it might also be gigabytes. So for 1GB of free space, it should use a quantity of 1 but your code would use a quantity of 1000. Doing this properly sounds like quite a lot of work, and the appropriate Android framework methods that cut between measure and quantity are annotated `@hide`. Hmm...